aws_lambda_builders/actions/python_pip/package.py

```diff
@@ -0,0 +1,582 @@
+import sys
+import hashlib
+import inspect
```

Comment:
hash lib and inspect are not used in the module.  I know it's nitpicky but I'm a nitpicker 8^)